env/dfctl

```diff
@@ -33,7 +33,7 @@ fi
 
 if [ "$1" == "test" ]
 then
-    cd test; pytest
+    cd test; pytest-3
```

Comment:
unrelated but ok

Reply:
I was supposed to be a local change, must have added it to repo accidentally. Sry
```suggestion
    cd test; pytest
```